2-mandatory/9-cheap-diner.js

```diff
@@ -19,7 +19,23 @@ chosenMeal(emptyArray)
 Should give the answer "Nothing :("
 **/
 
-function chooseMeal(mealArray) {}
+function chooseMeal(mealArray) {
+let emptyArray = []
+
+if(!mealArray.length) {
+  return "Nothing :("; 
+}
+
+if(mealArray.length === 1) {
+return mealArray[0].name; 
+}
+
+if(mealArray.length === 2) {
```

Comment:
How about if mealArray.length  > 2?I think the  code should work for any array of Meal